ral-facilities/operationsgateway
# DSEGOG-9 user can remove columns with close icon and middle mouse click

## Description
The user is now able to more easily remove columns from display. A black cross icon is displayed in each column header which the user can click to remove the column. I also added in the ability for the user to click the entire column header with the middle mouse button as an alternative method for removing the column from display. This mimics the behaviour of popular web browsers, where a user can middle click a tab to close it. This may be a familiar action for some users and speed up their interaction with the site.

## Testing instructions
Try removing columns using both methods and test that the functionality works as expected. Try this across multiple pages to double check it's working. Also make sure that, when a column is closed, its checkbox is unticked at the bottom of the page.


## Agile board tracking
Closes [DSEGOG-9](https://stfc.atlassian.net/jira/software/projects/DSEGOG/boards/22?selectedIssue=DSEGOG-9)
